Automattic/sensei
# Fix links in email previews

Resolves #6680.

## Proposed Changes
- Updates the CTA links in email previews to stay on the same page when clicked.
- Changes the site title so that it's no longer a link. I don't think it was a requirement to make it a link, so the easiest solution was just to remove it.

Note: I'm not sure why the footer link isn't clickable, but I didn't investigate it as it's not urgent.

## Testing Instructions


1. Preview each email.
2. Check that you can't click on the site title.
3. Click on the button.
4. You should stay on the current page.

## Pre-Merge Checklist
